scripts/index_exists.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
This duplicates a large amount of  `create_index.sh`. 

With getopts you could have a single script with different functions that execute dependent on the flags set.

Reply:
I think I'll leave it for now and see how many more similar scripts I need to make, if there is more of the same then I'll move them into a single script.